tensorflow/c/c_api.h

```diff
@@ -1019,6 +1019,14 @@ TF_CAPI_EXPORT extern void TF_GraphImportGraphDef(
     TF_Graph* graph, const TF_Buffer* graph_def,
     const TF_ImportGraphDefOptions* options, TF_Status* status);
 
+// Import the graph serialized in `graph_def` into `graph`.
+// Convenience function for when no results are needed.
+// Optionally try to bind resulted graph to given device ('/gpu:1' for example)
+void TF_GraphImportGraphDefWithDevice(TF_Graph* graph, const TF_Buffer* graph_def,
```

Comment:
Instead of adding a new function, I'd incorporate this binding into `TF_ImportGraphDefOptions`